core/assets/jsons/level.json

```diff
@@ -19,7 +19,7 @@
     "debugopacity":	192
   },
   "exit":             {
-    "pos":            [ 1.5, 10.35],
+    "pos":            [ 1.5, 5.35],
```

Comment:
Why?

Reply:
That was so I could test where the exit was when it was originally broken. Forgot I had changed it, can change back